lightly/active_learning/scorers/detection.py

```diff
@@ -193,17 +250,27 @@ def calculate_scores(self) -> Dict[str, np.ndarray]:
             to the scores (as a single-dimensional numpy array).
         """
         scores = dict()
-        scores['object-frequency'] = self._get_object_frequency()
-        scores['prediction-margin'] = self._get_prediction_margin()
+        scores_with_names = [
+            self._get_object_frequency(),
+            self._get_objectness_least_confidence()
+        ]
+        for score, score_name in scores_with_names:
+            score = np.nan_to_num(score)
+            scores[score_name] = score
+
+        # add classification scores
+        scores_dict_classification = \
+            _reduce_classification_scores_over_boxes(model_output=self.model_output)
+        for score_name, score in scores_dict_classification.items():
+            scores[f"classification_{score_name}"] = score
+
         return scores
 
     def _get_object_frequency(self):
-        scores = _object_frequency(
+        return _object_frequency(
             self.model_output,
             self.config['frequency_penalty'],
             self.config['min_score'])
```

Comment:
In the classification scorer you return the name of the score in this function and in the detection scorer you return it in the other - be consistent